packages/tailwind/modules/colors.ts

```diff
@@ -0,0 +1,51 @@
+type VuesticColorsList = Record<string, string>
+
+/**
+ * @param colors tailwind css colors options
+ * @returns vuestic ui colors options
+ */
+export const convertColors = (colors: Record<string, string | Record<string, string>>): VuesticColorsList => {
+  // system and deprecated by tailwind
+  const ignoredColors = ['inherit', 'current', 'transparent', 'coolGray', 'blueGray', 'trueGray', 'warmGray', 'lightBlue']
+  const result = {} as VuesticColorsList
+
+  for (const prop in colors) {
+    if (!ignoredColors.includes(prop)) {
```

Comment:
Can be this filtered before the `for (const prop in colors) {}` loop?

Reply:
I've tried:

<img width="961" alt="image" src="https://user-images.githubusercontent.com/64714442/205606721-441cbb81-2214-47a5-837f-c3e0b80b3e91.png">

But in this case (probably because of another way of accessing values) we gain Tailwind deprecated warnings:

<img width="644" alt="image" src="https://user-images.githubusercontent.com/64714442/205607419-da0caa99-8275-4c1b-92af-89c55a30ebe3.png">

We still can implement it this way:

<img width="307" alt="image" src="https://user-images.githubusercontent.com/64714442/205607241-d482b07b-ff63-4ef9-b07e-4a2c34a1495b.png">

But it's much more complicated than initial variant. So, let's it leave untouched :)